libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationNotificationProvider.java

```diff
@@ -34,8 +34,8 @@ void shutdown(Context applicationContext) {
 
   private NavigationNotification buildNotificationFrom(Context applicationContext, MapboxNavigation mapboxNavigation) {
     MapboxNavigationOptions options = mapboxNavigation.options();
-    if (options.navigationNotification() != null) {
-      return options.navigationNotification();
+    if (options.navigationNotification != null) {
```

Comment:
Can we use the let and elvis operator here

Reply:
👍